animals/templates/fullanimallist.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block pagetitle %}Current Animals{% endblock pagetitle %}
+
+{% block headertitle %}{% endblock headertitle %}
+
+{% block headersubtitle %}{% endblock headersubtitle %}
+
+{% block contenttitle %}
+<h2 class="section-title">Current animals <span>that need some help.</span></h2>
+<p>The list is below.</p>
+{% endblock contenttitle %}
+
+{% block bodycontent %}
+{% for a in animals %}
```

Comment:
usually we just use the singular form of the word to prevent any confusion later on. Not a need to change,  but just a thought for later on.

Reply:
Noted 👍